CMSgov/easi-app
# Enable Rich text (FE) - IT Gov

# EASI-3118

## Changes and Description

Rte fields for the rest of the IT Gov Admin Actions areas



## How to test this change

- Clean and seed so that some requests and admin notes are prefilled
- Go through all the admin action options and fill out the forms
- Check that the HTML content is displayed correctly in emails and the notes list
